### PR TITLE
Nginx Proxy attach requests

### PR DESCRIPTION
**Pull request by jeffbaier**
This pull request implements https://github.com/ehazlett/shipyard/issues/47

**Reminder:** there is a bug in node.js/hipache that is swallowing websockets. We either need to merge https://github.com/ehazlett/shipyard/pull/85 or close that pull request and I will add the necessary changes to this pull request.

### CODE REVIEW

File: .docker/run.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ DATABASES = {
     }
 }
 EOF
-git pull origin master
+git pull origin master --rebase
```
Comment (ehazlett) [inline .docker/run.sh:29]: Why the rebase?  Should we do something like git pull --ff-only origin master?

Comment (jeffbaier) [inline .docker/run.sh:29]: I think that would be the same thing. A pull --rebase is like doing a fetch then a fast-forward merge. I'm OK with whatever as long as it doesn't pop the editor. 

Maybe the better alternative is to add an env to not run that git pull command.

**Comment by adamjimenez** [comment]
what happened to the nginx-proxy?

**Comment by ehazlett** [comment]
Attach was removed in v2. Shipyard v3 no longer requires an external proxy
to use the browser console.

**Pull request merged**